doc/source/troubleshooting.rst

```diff
@@ -109,3 +109,15 @@ real_robot environments. This is also explained in the `Action Designator Exampl
    with simulated_robot:
       NavigateAction([Pose()]).resolve().perform()
 
+Missing pr2_arm_kinematics
+==========================
+
```

Comment:
perhaps here should also be a similar thing for what you get when you have that problem. The process usually gives an error that can be put here